chrome/content/zotero/charsetMenu.js

```diff
@@ -23,6 +23,119 @@
     ***** END LICENSE BLOCK *****
 */
 
+
+// Pulled from Fx63
+const CHARSET_MENU_DATA = {
+	"pinnedCharsets": [
+		{
+			"label": "Unicode",
+			"value": "UTF-8"
+		},
+		{
+			"label": "Western",
+			"value": "windows-1252"
+		}
+	],
+	"otherCharsets": [
```

Comment:
I think we can just remove the Import Character Encoding setting, the `extensions.zotero.import.charset` pref, and `otherCharsets` here. To the extent that this made sense 15 years ago, which is debatable, it's almost certainly pointless now. I'm sure approximately no one ever changes this setting, and in the extremely unlikely event that it's necessary, someone could convert the file with a text editor. We could also probably auto-detect a charset using the new JSWindowActor I added for that if our initial test reveals it's not UTF-8.

For exports, we're already only displaying UTF-8, UTF-8 w/o BOM, and Western.

For this PR, let's take this stuff out, and I'll fix up the import stuff in translate_firefox.js separately.